tests/web/test_web_about_openstax_tutor.py

```diff
@@ -0,0 +1,165 @@
+
+@test_case('')
+@web
+def test_view_about_tutor(web_base_url, selenium):
+    """Tests ability to view about OpenStax tutor page."""
+    
+    # GIVEN: On the OpenStax homepage
+
+    # WHEN: Hover over "Technology"
+
+    # AND: Click on "About OpenStax Tutor"
```

Comment:
`GIVEN` doesn't need a blank line before it; likewise, the `AND` is a part of the `WHEN` clause and doesn't need a blank line between them.